lib/reverse_sentence.rb

```diff
@@ -1,6 +1,41 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n)
+# Space complexity: O(1)
 def reverse_sentence(my_sentence)
```

Comment:
👍  This works and I like the helper method, but see my notes below on space complexity.